src/api/donations/dto/create-donation.input.ts

```diff
@@ -2,6 +2,9 @@ import { InputType, Field, Int } from '@nestjs/graphql';
 
 @InputType()
 export class CreateDonationInput {
+  @Field(() => Int)
```

Comment:
This field should not be here. The donor is more like an internal model to normalize it from donations and subscriptions. A user (using the api) will never create a donor and then a donation.